packages/backend-tasks/api-report.md

```diff
@@ -32,6 +33,12 @@ export type TaskFunction =
   | ((abortSignal: AbortSignal) => void | Promise<void>)
   | (() => void | Promise<void>);
 
+// @public
+export type TaskIds = {
+  global: string[];
+  local: string[];
+};
```

Comment:
Is there any particular reason to keep these types of IDs separate? From a retriggering standpoint there's no difference.

Reply:
That's in my opinion possible issue as well since the local and global task might have same id. Not sure if this should be allowed in the first place?

Reply:
Probably not, even though we don't check now. Assume for now that they don't collide.